app/src/main/java/com/artemzin/qualitymatters/models/PicassoBitmapClient.java

```diff
@@ -0,0 +1,21 @@
+package com.artemzin.qualitymatters.models;
+
+import android.widget.ImageView;
+import com.squareup.picasso.Picasso;
+
+import javax.inject.Inject;
+
+public class PicassoBitmapClient implements NetworkBitmapClient {
+
+  private final Picasso picasso;
+
+  @Inject
+  public PicassoBitmapClient(final Picasso picasso) {
+    this.picasso = picasso;
+  }
+
+  @Override
+  public void downloadInto(String url, ImageView imageView) {
+    picasso.load(url).fit().centerCrop().into(imageView);
```

Comment:
Yep, It is not okay for generic use case. In a real application, we should have several classes like this one, and use them depend on specific use case. But I think it's okay for a sample application like this. Should we add some comments for clarify?